src/pages/docs/deprecations/index.md

```diff
@@ -23,6 +23,17 @@ Deprecations are subject to change in detail or timeframe. If you need help asse
 
 ## Deprecations for 2023.2
```

Comment:
Should these be for 2023.3?

Reply:
Yeh, I think they all should. We just missed the cutover for `.2` (on purpose).